src/jit/flowgraph.cpp

```diff
@@ -22995,6 +22995,7 @@ void Compiler::fgRemoveEmptyTry()
                     fgRemoveStmt(block, finallyRet);
                     block->bbJumpKind = BBJ_ALWAYS;
                     block->bbJumpDest = continuation;
+                    fgAddRefPred(continuation, block);
                 }
```

Comment:
Yes, this was a fix necessitated by this change.

From what I can tell all the ref count updates are accurate right now (with the exception of the finally entry block, which is a special case). It would seem odd to deliberately leave them inaccurate.